roms_tools/setup/river_forcing.py

```diff
@@ -304,7 +304,7 @@ def _create_river_forcing(self, data):
         )
         tracer_data[:, 0, :] = 17.0
         tracer_data[:, 1, :] = 1.0
-        tracer_data[:, 2:, :] = 0.0
+        tracer_data[:, 2:, :] = 10.0
```

Comment:
It looks like you haven't added the BGC values here in the actual code (but just in the tests below).